app/models/case/base.rb

```diff
@@ -171,6 +171,20 @@ def self.searchable_document_tsvector
     )
   }
 
+  scope :partial_case, -> {
+    where(
+      "properties->>'is_partial_case'::text = ? ",
+      true.to_s
+    )
+  }
+
+  scope :not_partial_case, -> {
+    where(
```

Comment:
could this not just be if !true? rather than checking both false and null.

Reply:
does't work, != 'false' or <>'false' doesn't include the cases which doesn't have this field. This is why I wrote this way